be/src/exec/es/es_scan_reader.h

```diff
@@ -73,6 +76,8 @@ class ESScanReader {
     std::string _scroll_keep_alive;
     // timeout for es http connetion
     int _http_timeout_ms;
+
+    bool _exactly_once;
```

Comment:
1. give it a initial value
2. add more comments to explain in what scenario is this true

Reply:
1.  _exactly_once will be inited on the ESScanReader ctor
2. OK